service/src/main/java/org/mskcc/smile/service/impl/ResearchMessageHandlingServiceImpl.java

```diff
@@ -118,20 +118,7 @@ public void run() {
                 try {
                     SmileRequest request = requestQueue.poll(100, TimeUnit.MILLISECONDS);
                     if (request != null) {
-                        SmileRequest existingRequest =
-                                requestService.getSmileRequestById(request.getIgoRequestId());
-                        // save new request to database
-                        if (existingRequest == null) {
-                            LOG.info("Persisting new request: " + request.getIgoRequestId());
-                            requestService.saveRequest(request);
-                        } else {
-                            // request-service and sample-service methods will check for updates and persist
-                            // them if applicable (including patient swapping)
-                            requestService.updateRequestMetadata(request.getLatestRequestMetadata());
-                            for (SmileSample sample : request.getSmileSampleList()) {
-                                sampleService.saveSmileSample(sample);
-                            }
-                        }
```

Comment:
we still need this don't we? 

in particular [l132](https://github.com/mskcc/smile-server/pull/711/files#diff-942bee42f519ac33f382af4986299c80caaf461b0fc1158060ca1d59a08794a8L132) where we're letting the sample service handle updates for samples..